core/src/test/java/de/jplag/ParallelComparisonTest.java

```diff
@@ -1,28 +1,25 @@
 package de.jplag;
 
-import static de.jplag.strategy.ComparisonMode.PARALLEL;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
+import java.util.List;
 import java.util.Optional;
 
 import org.junit.jupiter.api.Test;
 
+import de.jplag.exceptions.BasecodeException;
 import de.jplag.exceptions.ExitException;
-import de.jplag.strategy.ParallelComparisonStrategy;
 
-/**
- * Currently just a copy of {@link NormalComparisonTest} but for the {@link ParallelComparisonStrategy}. // TODO TS:
- * de-duplicate this
- * @author Timur Saglam
- */
 public class ParallelComparisonTest extends TestBase {
```

Comment:
Now that default is parallel, we can maybe just name this class `DefaultComparisonTest`. In the future, we should split it by test case semantics, e.g. a single class for all multi-root tests, as we did for basecode tests.

Reply:
I think I will just add a note to the release board and merge this PR!